14_more_sinatra/badreads/app/views/genres/new.erb

```diff
@@ -0,0 +1,11 @@
+<form action="/genres" method="post">
+  <input type="text" name="name" placeholder="Name" />
+  <div>
+    <% @books.each do |b| %>
+    <input type="checkbox" id="<%= b.id %>" name="book_ids[]" value="<%= b.id %>" />
```

Comment:
This looks similar to what we did with radio buttons in `books/new.erb`, except Rack makes our life easier by letting us name the checkboxes `book_ids[]` and pass an `id` as each value. See `genres_controller` to understand how this works on the other side.